src/todo/todo.controller.ts

```diff
@@ -19,22 +22,45 @@ export class TodoController {
   constructor(private readonly _todoService: TodoService) {}
 
   @Post()
-  create(@Body() todo: CreateTodoDto, @User() user: { id: string }) {
+  create(
+    @Body() todo: CreateTodoDto,
+    @User() user: { id: string },
+  ): Promise<Todo> {
     return this._todoService.create({ title: todo.title, userId: user.id });
   }
 
   @Get()
-  getTodoList(@User() user: { id: string }) {
-    return this._todoService.findAll(user.id);
+  getTodoList(
+    @User() user: { id: string },
+    @Query() filterDto: FilterTodoDto,
+  ): Promise<Todo[]> {
+    return this._todoService.findAll({ userId: user.id, filter: filterDto });
   }
 
   @Get('/:todoId')
-  getTaskById(@User() user: { id: string }, @Param('todoId') todoId: string) {
+  getTodoById(
+    @User() user: { id: string },
+    @Param('todoId') todoId: string,
+  ): Promise<Todo> {
     return this._todoService.findOne({ userId: user.id, todoId: todoId });
   }
 
   @Delete('/delete/:id')
```

Comment:
correct standard path -> DETELE '/:id'

Reply:
I have removed it but I think I reverted the code change when I fixed the conflict